release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -1934,6 +1934,20 @@ For more information, see xref:../networking/metallb/metallb-upgrading-operator.
 * There is currently a known issue in the *Add Helm Chart Repositories* form to extend the Developer Catalog of a project. The *Quick Start* guides shows that you can add the `ProjectHelmChartRepository` CR in the desired namespace whereas it does not mention that to perform this you need permission from the kubeadmin. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2054197[BZ#2054197])
 * There is currently a known issue. If you create an instance of `ProjectHelmChartRepository` custom resource (CR) that uses TLS verification, you cannot list the repositories and perform Helm-related operations. There is currently no workaround for this issue. (link:https://issues.redhat.com/browse/HELM-343[*HELM-343*])
 
+* When running {product-title} on bare-metal IBM Power, there is a known issue that the Petitboot bootloader is unable to populate boot configurations for some {op-system} live images. In such cases, after booting nodes with PXE to install {op-system}, the expected live image disk configuration might not be visible upon reboot.
```

Comment:
```suggestion
* When running {product-title} on bare-metal IBM Power, there is a known issue that the Petitboot bootloader is unable to populate boot configurations for some {op-system} live images. In such cases, after booting nodes with PXE to install {op-system}, the expected live image disk configuration might not be visible.
```
I believe `booting nodes` implies the reboot. Purely a suggestion. Not sure if removing `upon reboot` changes the technical accuracy.